code/game/objects/structures/plasticflaps.dm

```diff
@@ -31,12 +31,17 @@
 	var/action = anchored ? "unscrews [src] from" : "screws [src] to"
 	var/uraction = anchored ? "unscrew [src] from " : "screw [src] to"
```

Comment:
I just realized this has an extra space at the end--that's not your bug, but it's easy enough to fix and *does* impact your code.

```suggestion
	var/uraction = anchored ? "unscrew [src] from" : "screw [src] to"
```